### DISCUSSION TITLE
Is there a way to setup invalidation on API/endpoint creation?

### DISCUSSION THREAD

**Discussion by fmartins-andre** [ecyrbe/zodios#479] [Q&A]
In RTK Query is easy to set what to invalidate at the endpoint declaration scope. Is there a way to do the same with Zodios?

**Comment by ecyrbe** [comment]
Hello,
No, invalidation is not declarative, it should be handled like react-query does it.
Indeed zodios is endpoint oriented (opposed to resource oriented like RTK or GraphQL) to allow any api out there that don't conform to rest strictly.
But i'm making it possible to declare ressources in this library here :
https://github.com/ts-api-spec/typematic

I'm planning to make next version of zodios use it. And this will unlock such feature in the future.

**Comment by ecyrbe** [comment]
Converting to a discussion.
